src/basic-syntax/scalar-types.md

```diff
@@ -16,3 +15,29 @@ The types have widths as follows:
 * `isize` and `usize` are the width of a pointer,
 * `char` is 32 bit wide,
 * `bool` is 8 bit wide.
+
+<details>
+
+There are a few syntaxes which are not shown above:
+
+- Raw strings allow you to create a `&str` value with escapes disabled: `r"\n"
+  == "\\\\n"`. You can embed double-quotes by using an equal amount of `#` on
```

Comment:
```suggestion
  == "\\n"`. You can embed double-quotes by using an equal amount of `#` on
```

I think.

Reply:
You are actually right according to the Commonmark spec: https://spec.commonmark.org/0.30/#example-338. However, I looked at the page and this is what I see with the weird number of backslashes:

![image](https://user-images.githubusercontent.com/89623/234950019-cb9f3efa-aeb8-4bb6-80c6-d1dad7b9a9e4.png)

I think there is a bug somewhere. I briefly tested https://github.com/raphlinus/pulldown-cmark and it handles the backticks correctly from what I can tell. So perhaps they get mangled further down the line.

Reply:
I tried disabling our preprocessors and the result is the same. I filed https://github.com/rust-lang/mdBook/issues/2079 to track this.